dataland-frontend/tests/e2e/specs/low/general/Datatables.ts

```diff
@@ -0,0 +1,18 @@
+describe('Datables test suite', () => {
+    it('Company Search by Name', () => {
```

Comment:
Kannst du die Beschreibung hier noch sprechender machen? 
z.B.: "Search for all companies containin 'a' and verify that results are paginated, only first 100 are shown"

Reply:
die beschreibung wurde angepasst